dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -80,6 +99,13 @@ public void nestedRecordsTest() throws Exception {
     public void convertGroupsRecordsTest() throws Exception {
         // TODO See section 5.4 of the Bean Validation specification for groups
         // and section 5.4.5 for group conversion
+
+        Registration reg = new Registration("x1asas", false);
+        Set<ConstraintViolation<Registration>> constraintViolations = validator.validate(reg);;
+        assertTrue("Record Registration should have validated with no violations", constraintViolations.size() == 0);
+        Company cmp2 = new Company(" sds", reg);
+        Set<ConstraintViolation<Company>> constraintViolations2 = validator.validate(cmp2, RegistrationChecks.class);;
+        assertTrue("Record Company should have validated with no violations after conversion", constraintViolations2.size() == 2);
```

Comment:
This message should be updated to indicate that 2 violations are expected.

Reply:
updated